### DISCUSSION TITLE
Choose dataportal URL on login WPF app

### DISCUSSION THREAD

**Discussion by BlagoCuljak** [MarimerLLC/csla#1661] [Questions]
Hi all,
I'm having real issue with one task that I want to implement. On login of my WPF app I would like to choose what dp url it connects to.

I would store multiple urls in .json and try to use one of them.

Did someone had a shot at this?

**Comment by rockfordlhotka** [comment]
You can change the default URL at any time by calling the CSLA config subsystem.

From memory 

’’’
Csla.Configuration.Configure()
  .DataPortal().DefaultProxy(type, url);
’’’

**Reply by EricNgo1972** [reply]
I use an Azure table instead of .json. 

It helps when you update the list of  DP URL.
